op-e2e/actions/l2_proposer.go

```diff
@@ -31,6 +31,7 @@ type ProposerCfg struct {
 	OutputOracleAddr       *common.Address
 	DisputeGameFactoryAddr *common.Address
 	ProposalInterval       time.Duration
+	ProposalRetryInterval  time.Duration
```

Comment:
Maybe we rename this `OutputRetryInterval` because it really describes retrying getting the output, not submitting the proposal. Same for the flag etc.